src/base/static/state/ducks/app-config.js

```diff
@@ -5,7 +5,7 @@ export const appConfigSelector = state => {
 };
 
```

Comment:
All changes in ducks below are either meant to prevent null initial states, or to normalize `load` terminology.